replicated_kv_store/kv_store/restaccess_key_value.go

```diff
@@ -41,10 +42,15 @@ func (kv *store) KvstoreHandler(w http.ResponseWriter, r *http.Request) {
 //handles all post requests
 func (kv *store) PostHandler(w http.ResponseWriter, r *http.Request) {
 
-	// if r.Method != "POST" {
-	// 	http.Error(w, "Method is not supported.", http.StatusNotFound)
-	// 	return
-	// }
+	log.Printf("\nPOST request received\n")
+
+	if r.Method != "POST" {
```

Comment:
I'm not sure if these request type checks in the HTTP handler functions are required, because in main.go while registering these handlers, the Method has also been defined.